kibbeh/src/modules/room/chat/RoomChatInput.tsx

```diff
@@ -113,86 +113,61 @@ export const RoomChatInput: React.FC<ChatInputProps> = ({ users }) => {
-        <Picker
-          set="apple"
-          onSelect={(emoji: CustomEmote) => {
-            position =
-              (position === 0
-                ? inputRef!.current!.selectionStart
-                : position + 2) || 0;
-
-            const newMsg = [
-              message.slice(0, position),
-              "native" in emoji
-                ? emoji.native
-                : (message.endsWith(" ") ? "" : " ") +
-                  (emoji.colons || "") +
+        <div className={`mb-1`}>
+          <EmojiPicker
+            emojiSet={customEmojis}
+            onEmojiSelect={(emoji) => {
+              position =
+                (position === 0
+                  ? inputRef!.current!.selectionStart
+                  : position + 2) || 0;
+
+              const newMsg = [
+                message.slice(0, position),
+                (message.endsWith(" ") ? "" : " ") +
+                  (`:${emoji.short_names[0]}:` || "") +
                   " ",
-              message.slice(position),
-            ].join("");
-            setMessage(newMsg);
-          }}
-          style={{
-            position: "relative",
-            width: "100%",
-            minWidth: "278px",
-            right: 0,
-            overflowY: "hidden",
-            outline: "none",
-            alignSelf: "flex-end",
-            margin: "0 0 8px 0",
-          }}
-          sheetSize={32}
-          theme="dark"
-          custom={customEmojis}
-          emojiTooltip={true}
-          showPreview={false}
-          showSkinTones={false}
-          i18n={{
-            search: t("modules.roomChat.search"),
-            categories: {
-              search: t("modules.roomChat.searchResults"),
-              recent: t("modules.roomChat.recent"),
-            },
-          }}
-        />
+                message.slice(position),
+              ].join("");
+              setMessage(newMsg);
+            }}
+          />
+        </div>
       ) : null}
       <div className="flex items-stretch">
-        <div className="flex-1 mr-2 lg:mr-0 items-end">
+        <div className="flex-1 mr-2 lg:mr-0 items-center bg-primary-700 rounded-8 flex-row">
           <Input
             maxLength={512}
             placeholder={t("modules.roomChat.sendMessage")}
             value={message}
             onChange={(e) => setMessage(e.target.value)}
-            className={`bg-primary-700`}
+            id="room-chat-input"
+            transparent={true}
             ref={inputRef}
             autoComplete="off"
             onKeyDown={navigateThroughQueriedUsers}
             onFocus={() => {
               setIsEmoji(false);
               position = 0;
             }}
-            id="room-chat-input"
           />
-          {/* <div
+          <div
             style={{
               color: "rgb(167, 167, 167)",
               display: "flex",
               marginRight: 13,
-              marginTop: -35,
               flexDirection: "row-reverse",
```

Comment:
can be set using classname

Reply:
also instead of using a custom color, use a tailwind color

Reply:
I didn't even write that code it's probably patched in from `kofta`. I'll fix it anyways